pkg/crd/operator/v1alpha1/configuration.go

```diff
@@ -167,6 +167,8 @@ type KubermaticUserClusterConfiguration struct {
 	DisableAPIServerEndpointReconciling bool `json:"disableApiserverEndpointReconciling,omitempty"`
 	// EtcdVolumeSize configures the volume size to use for each etcd pod inside user clusters.
 	EtcdVolumeSize string `json:"etcdVolumeSize,omitempty"`
+	// EtcdReplicasNumber configures the number of replicas to use by default for each etcd pod inside user clusters.
+	EtcdReplicasNumber int `json:"etcdReplicasNumber,omitempty"`
```

Comment:
Why did you call this `ReplicasNUMBER`, whereas we just call it `Replicas` everywhere else? It feels a bit over-specific and incocsitent w.r.t. e.g. the following line: APIServerReplicas.